Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -100,23 +89,45 @@ lemma inv_rnDeriv [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪ ν) (hνμ :
     (by filter_upwards [hνμ.ae_le (rnDeriv_pos hμν)] with x hx using hx.ne')
     (rnDeriv_ne_top _ _)]
 
+lemma set_lintegral_rnDeriv_eq_withDensity {s : Set α} (hs : MeasurableSet s) :
+    ∫⁻ x in s, μ.rnDeriv ν x ∂ν = ν.withDensity (μ.rnDeriv ν) s :=
+  (withDensity_apply _ hs).symm
```

Comment:
Do you really need this lemma? It looks like a simple specialization of `withDensity_apply`, that has nothing to do with the fact that you use a Radon-Nikodym derivative.

Reply:
I'm not sure. On one hand, it's obviously just a special case of `withDensity_apply` and is not really needed. Furthermore, most often if we rewrite that integral it's with `set_lintegral_rnDeriv `, not with that one. On the other hand I added it because several times it took me a few seconds to remember the name of `withDensity_apply` (because there is no withDensity on the left hand side here). Also it makes the API for lintegral match the API of integral (where I find the corresponding lemma useful).
So I don't know. Keeping or removing are both fine by me.

Reply:
I removed it.